core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,10 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
+        'StatesToQuestionsService', function($scope, SimpleEditorManagerService,
             explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+            explorationWarningsService, QuestionIdService,
+            StatesToQuestionsService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+          $scope.modalWrapperVisible = false;
```

Comment:
That's not a modal; that's a dropdown. Why don't you use a standard bootstrap dropdown like we do in the About menu in the navbar?